Dockerfile

```diff
@@ -10,23 +10,28 @@ ENV VERSION ${VERSION}
 RUN envsubst '${VERSION}' < /web/src/utils/config.js > /web/src/utils/config.js.subst && mv /web/src/utils/config.js.subst /web/src/utils/config.js
 RUN npm run build
 
-FROM nginx:1.20.1-alpine
-RUN apk update && \
-    apk upgrade && \
-    apk add curl && \
-    apk add bash && \
-    apk add gettext && \
-    apk add shadow && \
-    rm -rf /var/cache/apk/*
+FROM registry.suse.com/bci/bci-base:15.3
+
+RUN zypper -n install curl libxml2 bash gettext shadow && \
+    rm -f /bin/sh && ln -s /bin/bash /bin/sh
+
+RUN zypper addrepo --gpgcheck --type yum --refresh --check \
+    'http://nginx.org/packages/sles/$releasever_major' nginx-stable && \
+	curl -o /tmp/nginx_signing.key https://nginx.org/keys/nginx_signing.key && \
+    [[ `gpg --show-keys /tmp/nginx_signing.key  | grep -o "573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62"` ]] && \
+    rpm --import /tmp/nginx_signing.key && \
+    zypper ref && \
+    zypper -n install nginx && \
+    rm -rf /var/cache/zypp/*
+
 RUN mkdir -p web/dist
 WORKDIR /web
+
 COPY --from=builder /web/dist /web/dist
 COPY --from=builder /web/nginx.conf.template /etc/nginx/nginx.conf.template
 
+EXPOSE 8000
 ENV LONGHORN_MANAGER_IP http://localhost:9500
 ENV LONGHORN_UI_PORT 8000
 
-RUN useradd -r -u 1000 longhorn
```

Comment:
Why we removed this? This is for security hardened.

Reply:
Nginx will start using `nginx`  user

Reply:
It looks you are using nginx rpm package to install, but unfortunately nginx doesn't support arm64. Also, this will be a problem when we support other architectures. Probably we should just use the official niginx image instead.

Reply:
Sure, I'll close the PR, and we will continue using Nginx official image.

Reply:
https://github.com/longhorn/longhorn-ui/pull/510#issuecomment-1103325818

Let's see if we can use Nginx from the built-in repo in the BCI image directly.